src/components/FeatureViews/Availability/SubmitNotice.js

```diff
@@ -90,7 +91,26 @@ export default function SubmitNotice() {
 		});
 	}
 
-	return (<Grid container style={{ paddingLeft: "8rem" }} direction="row">
+	const disablePrevDates = (startDate) => {
+		const startSeconds = Date.parse(startDate);
+		return (date) => {
+			return Date.parse(date) < startSeconds;
+		}
```

Comment:
use moment's isBefore function to do this: https://momentjscom.readthedocs.io/en/latest/moment/05-query/01-is-before/

also rename + resimplify ~ you're checking to see if a given date is after today.
```
const isUpcomingDate = (startDate) => moment().isBefore(moment(startDate))
```

Reply:
1. we are using prop in OOOConfirmation. 

```
<Grid item xs={12} alignItems="center" style={{ marginTop: "5vh" }}>
                <Button 
                    className={classes.cancelbutton}
                    variant="outlined" 
                    onClick={props.handleClose}>Close</Button>
            </Grid>
```

2. Renaming to isUpcomingDate makes it confusing. What this function is doing is checking if the day is before the current date. I will rename to ```checkIfDateBeforeToday```